string_functions.py

```diff
@@ -0,0 +1,33 @@
+#Enter a string value
+a = input("Enter a string value: ")
+
+#Print out the string value
+print(f"You've entered the string <{a}>")
+
+#Apply method lower to the string value
+b = str.lower(a)
```

Comment:
This works, but what I have asked you guys to do is do "b = a.lower()". Do the similar change to all methods.